src/main/java/AutomationPracticeLyrics/LostOnYouLyricsPage.java

```diff
@@ -0,0 +1,71 @@
+package AutomationPracticeLyrics;
+
+import Utils.WaitMethods;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+import java.util.*;
+
+public class LostOnYouLyricsPage {
+    @FindBy(xpath = "//pre[@id = 'lyric-body-text']")
+    private WebElement lostOnYouLyrics;
+
+    @FindBy(xpath = "//i[@class = 'fa fa-play']")
+    private List<WebElement> playButtons;
+
+    @FindBy(xpath = "//h1[@id='lyric-title-text']")
+    private WebElement lostOnYouTitle;
+
+    WaitMethods waitMethods = new WaitMethods();
+
+    public String getLyrics() {
+        this.waitMethods.isElementDisplayed(this.lostOnYouLyrics, 5);
+        return this.lostOnYouLyrics.getText();
+    }
+
+    public int countLetters(String lyrics) {
+        int count = 0;
+        for (int i = 0; i < lyrics.length(); i++) {
+            if (Character.isLetter(lyrics.charAt(i))) {
+                count++;
+            }
+        }
+        return count;
+    }
+
+    public int countOccurrences(String word, String lyrics) {
+        int count = (lyrics.split(word, -1).length) - 1;
+        return count;
+    }
+
+    public ArrayList<String> makeListFromLyrics (String lyrics) {
+        String[] words1 = lyrics.split(" ");
+        ArrayList<String> words = new ArrayList<String>(Arrays.asList(words1));
```

Comment:
Redundant. 
Go with inline variable so that you will write 1 line of code less :D

Reply:
Roger!